### PR TITLE
Integrate v4l2 improvements for the xdpzu7

### PR DESCRIPTION
**Pull request by MikeLooijmans**
This integrates Bob's work on V4L2 into the TLD

### CODE REVIEW

File: recipes-kernel/linux/topic-xilinx-kernel-patches.inc.orig [hunk 1]
```diff
@@ -0,0 +1,82 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/linux-xlnx:"
```
Comment (niekvanagt) [inline recipes-kernel/linux/topic-xilinx-kernel-patches.inc.orig]: I think this is checked in by accident. (.orig)

Comment (MikeLooijmans) [reply recipes-kernel/linux/topic-xilinx-kernel-patches.inc.orig]: Yeah, good catch

**Comment by marctopic** [comment]
Why is this review on github and not on bitbucket?
Can we not move meta-topic to bitbucket?

**Comment by MikeLooijmans** [comment]
> Why is this review on github and not on bitbucket?

because the repo is on github only

**Pull request merged**